mod/custom_index_widgets/start.php

```diff
@@ -8,7 +8,7 @@
 	 
     function custom_index_widgets_init() {
 
-if (elgg_is_active_plugin('gc_fedsearch_gsa') && ((!$gsa_usertest) && strcmp($gsa_agentstring,strtolower($_SERVER['HTTP_USER_AGENT'])) == 0) || strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') !== false ) {
+if (elgg_is_active_plugin('gc_fedsearch_gsa') && ((!$gsa_usertest) && strcmp($gsa_agentstring,strtolower($_SERVER['HTTP_USER_AGENT'])) == 0) && (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') !== false || strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'solr-crawler') === false) ) {
```

Comment:
Is this supposed to be  === here? It makes that part evaluate to true for regular browsers.

Reply:
If user-agent is gsa-crawler or user-agent is solr-crawler, do not display widgets